lxd/instance/drivers/driver_lxc.go

```diff
@@ -3958,6 +3958,13 @@ func (d *lxc) CGroupSet(key string, value string) error {
 
 // Update applies updated config.
 func (d *lxc) Update(args db.InstanceArgs, userRequested bool) error {
+	// Setup a new operation
+	op, err := operationlock.CreateWaitGet(d.Project(), d.Name(), operationlock.ActionUpdate, []operationlock.Action{operationlock.ActionRestore}, false, false)
```

Comment:
Why action restore here BTW?

Reply:
Because the Restore function calls Update() and we don't want that to deadlock.